tests/getCharacter.spec.js

```diff
@@ -49,7 +49,7 @@ describe('9 - Implemente os casos de teste da função `getCharacter`', () => {
     // Teste se a função retorna o objeto correto para o parâmetro 'Arya',
     // Teste se a função retorna o objeto correto para o parâmetro 'Brienne',
     // Teste se a função retorna o objeto correto para o parâmetro 'Melissandre',
-    // Teste se a função se os parâmetros não são Case Sensitive.
-    // Teste se ao passar um nome que não está na tabela, a função retorna undefined.
+    // Teste se os parâmetros não são Case Sensitive.
+    // Teste se, ao passar um nome que não está na tabela, a função retorna undefined.
```

Comment:
```suggestion
    // Teste se ao passar um nome que não está na tabela, a função retorna undefined.
```
Faz sentido esse ter virgula e outros acima não?